inputremapper/injection/macros/macro.py

```diff
@@ -482,9 +483,19 @@ def add_mouse(self, direction: str, speed: int):
         }[direction.lower()]
 
         async def task(handler: Callable):
-            resolved_speed = value * _resolve(speed, [int])
+            resolved_speed = _resolve(speed, [int])
+            resolved_accel = _resolve(accel, [float, None])
+
+            if resolved_accel:
+                current_speed = 0.0
+            else:
+                current_speed = resolved_speed
+
             while self.is_holding():
-                handler(EV_REL, code, resolved_speed)
+                if resolved_accel and current_speed < resolved_speed:
+                    current_speed += resolved_accel
+
+                handler(EV_REL, code, value * int(current_speed))
```

Comment:
```suggestion
                if resolved_accel and current_speed < resolved_speed:
                    current_speed += resolved_accel

                # To allow a macro of mouse(right, 1, 0.001) to move slowly, instead of
                # jumping to a speed of 1 eventually, progress the pointer by a pixel
                # by adding up the dither_remainder.
                dithered_speed = value * current_speed
                dither_remainder += dithered_speed - math.floor(dithered_speed)
                dithered_speed = math.floor(dithered_speed)

                if dither_remainder > 1:
                    dithered_speed += 1
                    dither_remainder -= 1

                handler(EV_REL, code, dithered_speed)
```

Reply:
Oh, and initialize `dither_remainder = 0` above the while loop

Reply:
I'm not sure if dither is the correct term though. Dither seems to be specifically about avoiding banding using noise, but this isn't using noise.

Reply:
Yeah, that makes sense. I had been considering something like this, but wasn't sure how much difference it would make in practice. Since you've brought it up as well, I've added an implementation of this idea.